### PR TITLE
Update release script

### PR DESCRIPTION
**Pull request by sestevens**
### Summary of changes

 - Update release script to clarify that pressing any key will cause the script to continue after pre-release checklist is shown.

 ### Checklist

 - [ ] ~Added a changelog entry~

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -148,7 +148,7 @@ namespace :release do
     say "* [ ] Tests (rake spec) are passing, manual verifications complete."
     say "* [ ] Email is composed and ready to send to braintree-sdk-announce@googlegroups.com"
 
-    abort(1) unless ask "Ready to release? "
+    abort(1) unless ask "Ready to release? Press any key to continue. "
```
Comment (scannillo) [inline Rakefile]: Is it a ton more work to have the `Y` and `N` keys programmed to continue or cancel here?

Comment (sestevens) [reply Rakefile]: Probably not a ton, but it would be some work. I'd say let's leave this as a future improvement 🙂

Comment (sestevens) [reply Rakefile]: @scannillo Are you good with this being merged?

Comment (scannillo) [reply Rakefile]: YES!

**Pull request merged**